plugins/woocommerce/src/Blocks/BlockTypes/ProductFilterPriceSlider.php

```diff
@@ -60,9 +60,10 @@ protected function render( $attributes, $content, $block ) {
 
 		$wrapper_attributes = get_block_wrapper_attributes(
 			array(
-				'class'       => esc_attr( $classes ),
-				'style'       => esc_attr( $style ),
-				'data-wp-key' => wp_unique_prefixed_id( $this->get_full_block_name() ),
+				'class'               => esc_attr( $classes ),
+				'style'               => esc_attr( $style ),
+				'data-wp-interactive' => 'woocommerce/product-filters',
+				'data-wp-key'         => wp_unique_prefixed_id( $this->get_full_block_name() ),
```

Comment:
Super super nitpick, don't hate for pointing it out such a detail but it feels like `data-wp-interactive` is the most important property and everywhere else it's at the top. The order here stands out 😅 

```suggestion
				'data-wp-interactive' => 'woocommerce/product-filters',
				'data-wp-key'         => wp_unique_prefixed_id( $this->get_full_block_name() ),
				'class'               => esc_attr( $classes ),
				'style'               => esc_attr( $style ),
```